.github/workflows/jgiven_android_build.yml

```diff
@@ -1,5 +1,5 @@
 name: "Build and test android plugin and projects"
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Is this a huge improvement to remove the push?

Reply:
Goot thing we're talking about this. That was only to avoid excessive builds during testing how the change affects renovate (Because with this change we should get better coverage of renovate for our dependencies)

Reply:
Then this should be enabled again in the final commit.